ui/hooks/snaps/useTransactionInsightSnaps.js

```diff
@@ -98,16 +84,12 @@ export function useTransactionInsightSnaps({
     };
   }, [
     transaction,
-    eagerFetching,
     chainId,
     origin,
     subjects,
     // TODO: Figure out how to improve this
     JSON.stringify(insightSnaps),
-    insightSnapId,
-    ///: BEGIN:ONLY_INCLUDE_IF(build-flask)
-    hasFetchedV2Insight,
-    ///: END:ONLY_INCLUDE_IF
+    hasFetchedInsight,
   ]);
 
   return { data, loading };
```

Comment:
I thought we agreed to add the warnings logic here? Now it's reverted?

Reply:
Yes! Need to revert this change. I was experimenting when things were breaking and didn't change back.